File: 01.ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P02_PassedOrFailed/P02_PassedOrFailed.cs
using System;

namespace P02_PassedOrFailed
{
    class P02_PassedOrFailed
    {
        static void Main(string[] args)
        {
            double grade = double.Parse(Console.ReadLine());
            Console.WriteLine((grade >= 3.00) ? "Passed!":"Failed!");
        }
    }
}

File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P02_PassedOrFailed.java
import java.util.Scanner;
public class P02_PassedOrFailed {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        double grade = Double.parseDouble(console.nextLine());
        System.out.println((grade >= 3.0) ? "Passed!":"Failed!");
    }
}
